packages/category-feed-luigis-box/src/Model/FeedItem/LuigisBoxCategoryFeedItemFacade.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\CategoryFeed\LuigisBoxBundle\Model\FeedItem;
+
+use Shopsys\CategoryFeed\LuigisBoxBundle\Model\Category\LuigisBoxCategoryRepository;
+use Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig;
+
+class LuigisBoxCategoryFeedItemFacade
+{
+    /**
+     * @param \Shopsys\CategoryFeed\LuigisBoxBundle\Model\Category\LuigisBoxCategoryRepository $luigisBoxCategoryRepository
+     * @param \Shopsys\CategoryFeed\LuigisBoxBundle\Model\FeedItem\LuigisBoxCategoryFeedItemFactory $luigisBoxFeedItemFactory
+     */
+    public function __construct(
+        protected readonly LuigisBoxCategoryRepository $luigisBoxCategoryRepository,
+        protected readonly LuigisBoxCategoryFeedItemFactory $luigisBoxFeedItemFactory,
+    ) {
+    }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig $domainConfig
+     * @param int|null $lastSeekId
+     * @param int $maxResults
+     * @return \Shopsys\CategoryFeed\LuigisBoxBundle\Model\FeedItem\LuigisBoxCategoryFeedItem[]|iterable
```

Comment:
I believe the proper annotation is:
```suggestion
     * @return iterable<int, \Shopsys\CategoryFeed\LuigisBoxBundle\Model\FeedItem\LuigisBoxCategoryFeedItem>
```

Reply:
(similarly in `LuigisBoxFeedItemFacade::getItems`)

btw, I do not want to be that guy, but IMHO the navigation is a little bit confusing with the naming:
- `LuigisBoxFeedItemFacade`
- `LuigisBoxCategoryFeedItemFacade`
- `LuigisBoxArticleFacade`

it is not obvious that the first one is for products (yes, it is the name of the bundle, but...) and the naming is not unified so I had a hard time finding the proper files by their names.

Reply:
I have introduced some naming changes to unify it

Reply:
thank you, now it is much better :pray: 

Two more things:
- could you please fix the iterable annotation in `LuigisBoxProductFeedItemFacade::getItems()`?
- after the classes renaming, some of the property and variable names do not match but I believe I can live with that and if you do not want to fix it, I understand :blush: